### PR TITLE
Remove 'LOG: ' from log

### LINKED ISSUE #1703: [Runtime] Remove `LOG: ` from logs
Currently, when contract write logs they get prefixed with `LOG: ` prefix which is unnecessary information, but something we have already designed nearlib to expect.

### PR DESCRIPTION
**Pull request by fckt**
fixes #1703

### CODE REVIEW

File: runtime/near-vm-logic/src/logic.rs [hunk 1]
```diff
@@ -1561,7 +1561,7 @@ impl<'a> VMLogic<'a> {
         let message = self.get_utf8_string(len, ptr)?;
         self.gas_counter.pay_base(log_base)?;
         self.gas_counter.pay_per_byte(log_byte, message.as_bytes().len() as u64)?;
-        self.logs.push(format!("LOG: {}", message));
+        self.logs.push(format!("{}", message));
```
Comment (evgenykuzyakov) [inline runtime/near-vm-logic/src/logic.rs]: don't need to format the message. It's already a string.

File: runtime/near-vm-logic/src/logic.rs [hunk 2]
```diff
@@ -1584,7 +1584,7 @@ impl<'a> VMLogic<'a> {
             log_byte,
             message.encode_utf16().count() as u64 * size_of::<u16>() as u64,
         )?;
-        self.logs.push(format!("LOG: {}", message));
+        self.logs.push(format!("{}", message));
```
Comment (evgenykuzyakov) [inline runtime/near-vm-logic/src/logic.rs]: same here

File: runtime/near-vm-logic/tests/test_miscs.rs [hunk 3]
```diff
@@ -17,10 +17,7 @@ fn test_valid_utf8() {
     let len = string_bytes.len() as u64;
     logic.log_utf8(len, string_bytes.as_ptr() as _).expect("Valid utf-8 string_bytes");
     let outcome = logic.outcome();
-    assert_eq!(
-        outcome.logs[0],
-        format!("LOG: {}", String::from_utf8(string_bytes.clone()).unwrap())
-    );
+    assert_eq!(outcome.logs[0], format!("{}", String::from_utf8(string_bytes.clone()).unwrap()));
```
Comment (evgenykuzyakov) [inline runtime/near-vm-logic/tests/test_miscs.rs]: and here

**Comment by nearmax** [comment]
See that CI is failing on this PR, you would have to create a PR to https://github.com/nearprotocol/nearlib/ and submit this in parallel.

**Comment by evgenykuzyakov** [comment]
Please remove remaining unnecessary `format` there

**Comment by nearmax** [comment]
Is it still blocked?

**Comment by fckt** [comment]
@nearmax 
right, we should bump the near version, see
https://github.com/nearprotocol/nearlib/pull/113/files#diff-e4b7944031f28254849e1f2cb31a64e4R138

**Comment by nearmax** [comment]
I have merged https://github.com/nearprotocol/nearcore/pull/1873 in so that we can build a new version.

**Comment by nearmax** [comment]
@fckt I have bumped runtime crates, but there is no need to bump near crate version, since it is already 0.4.10 in master and staging. I don't understand why current master and staging CI does not fail since nearlib change was already merged. I have asked question on Slack.

**Comment by nearmax** [comment]
Okay, the confusion was resolved. It was https://github.com/nearprotocol/nearlib/pull/150

It should be passing CI now.

**Pull request merged**